include/spblas/vendor/onemkl_sycl/mkl_allocator.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <sycl.hpp>
+
+namespace spblas {
+namespace mkl {
```

Comment:
I think we were using `__mkl` namespace almost everywhere else for vendor specific stuff.  Are we wanting to change that here ?

Reply:
I thought the functions in `__mkl` namespace for developers which should not be used by users, but user can use the function from `mkl` namespace.

Reply:
ahh,  ok I gues that makes sense as well.  lets leave it as is for now.